src/debugging/DockerDebugConfigurationProvider.ts

```diff
@@ -17,6 +17,12 @@ export interface DockerDebugConfiguration extends NetCoreDockerDebugConfiguratio
     platform?: DockerPlatform;
 }
 
+export interface DockerAttachConfiguration extends NetCoreDockerDebugConfiguration, NodeDockerDebugConfiguration {
+    debuggerFilePath?: string;

```

Comment:
This would have to be `debuggerPath` to match what is in package.json; else in there make it `debuggerFilePath`.

Reply:
Yes, Last minute I changed the property from debuggerFilePath to debuggerPath, but forgot to update in all places.